how-it-works/vipps-login-apps-howitworks.md

```diff
@@ -0,0 +1,21 @@
+<!-- START_METADATA
+---
+title: How Login works from apps
+sidebar_label: From apps 
+sidebar_position: 13
+description: How Login works from apps.
+pagination_next: null
+pagination_prev: null
+---
+END_METADATA -->
+
+# How Login works from apps
+
+<!-- START_COMMENT -->
+
+ℹ️ Please use the website:
+[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/docs/APIs/login-api).
+
+<!-- END_COMMENT -->
+
+Vipps Login app integrations uses the browser for performing logins. See [Vipps Login in browser: How It Works](vipps-login-api-howitworks.md).
```

Comment:
I think I would delete this page, unless you are going to add more to it in the future.

Reply:
It _is_ super short..? 🧐

Reply:
I think we could make things clearer for merchants if we actually did describe the complete flow here. It's only partially true that it's the same as the one we link too.